spec/services/idv/session_spec.rb

```diff
@@ -33,4 +33,37 @@
       end
     end
   end
+
+  describe '#complete_session' do
+    context 'with phone verifed by vendor' do
+      before do
+        subject.address_verification_mechanism = :phone
+        subject.phone_confirmation = true
+      end
+
+      context 'having completed OTP phone confirmation' do
+        before do
+          subject.params['phone_confirmed_at'] = Time.zone.now
+        end
+
+        it 'completes the user profile' do
+          allow(subject).to receive(:complete_profile)
+          subject.complete_session
+          expect(subject).to have_received(:complete_profile)
```

Comment:
Can we please add a blank line above the expectation? The style guide we follow in tests is to separate the test setup from the expectations with a blank line.